src/test/java/org/osiam/client/OsiamUserServiceTest.java

```diff
@@ -54,51 +66,51 @@ public void setUp() throws Exception {
     @Test(expected = IllegalArgumentException.class)
     public void id_is_null_by_getting_single_user_raises_exception() throws Exception {
         service.getUser(null, accessToken);
-
-        fail("Exception expected");
     }
 
     @Test(expected = NullPointerException.class)
     public void access_token_is_null_by_getting_single_user_raises_exception() throws Exception {
         service.getUser(searchedID, null);
-
-        fail("Exception expected");
     }
 
     @Test(expected = NullPointerException.class)
     public void access_token_is_null_by_getting_all_group_raises_exception() throws Exception {
         service.getAllUsers(null);
-
-        fail("Exception expected");
     }
 
     @Test(expected = NullPointerException.class)
     public void create_null_user_raises_exception() {
         service.createUser(null, accessToken);
-
-        fail("Exception excpected");
     }
 
     @Test(expected = NullPointerException.class)
     public void create_user_with_null_access_token_raises_exception() {
         User newUser = new User.Builder("irrelevant").build();
-
         service.createUser(newUser, null);
+    }
 
-        fail("Exception excpected");
+    @Test(expected = BadRequestException.class)
+    public void invalid_filter_generates_bad_request() {
```

Comment:
You can ignore single files or single issues. I ignored this one, but the commit status is still 'failed'. Anyways, I'll will merge it though.